modules/ksql-cluster/outputs.tf

```diff
@@ -3,9 +3,9 @@ output "api_version" {
   value       = confluent_ksql_cluster.ksqldb_cluster.api_version
 }
 
-output "http_endpoint" {
+output "rest_endpoint" {
```

Comment:
yes it's breaking change from confluent API

Reply:
oh you mean to put the breaking change in the PR?

Reply:
Means: We could fix the value in line 8, without needing to rename this output.

So you keep two outputs, one is `http_endpoint` and you add a new one named `rest_endpoint`. You can in both plug ksqldb_cluster.rest_endpoint, but do not remove the `http_endpoint` output so that the consumers can keep using that output, till they change to the new one, named `rest_endpoint`.

After consumers using new output `rest_endpoint`, then you can remove output `http_endpoint`. But of course, you could also rename as in the PR, just would break the consumers, which maybe is ok since we have only one, I believe.